R/bundle_embed.R

```diff
@@ -0,0 +1,24 @@
+#' @export
```

Comment:
I added some documentation templates yesterday that should make documenting this method pretty quick. Could you merge upstream and copy those over from another bundler? a la:

https://github.com/simonpcouch/bundle/blob/5d3c0216192b6b3a0497d0c51d802e4c326ebf45/R/bundle_lightgbm.R#L1-L13

Reply:
The upstream merge might conflict with my commenting out the `bundle.recipe` tomfoolery. Feel free to overwrite my changes there.